### DISCUSSION TITLE
How do I gain insight to similar documents?

### DISCUSSION THREAD

**Discussion by sonsuzluk0** [asreview/asreview#1217] [Q&A]
Hello!

I am sorry for my noob question. I have installed asreview Lab. I started it via _asreview lab_ in command line (Windows 10 Pro). and also created a project, trained (Oracle method) and finished the according project. But I do not see any option for acquiring relevant documents via the internet. There is no option where I can search for documents that are similar to my trained model. Do I miss anything? I only can train the AI, as far as I can see. 

Thanks.

**Comment by gimoAI** [comment]
Hi @sonsuzluk0,

The way asreview lab is designed, requires you to gather all the documents that you want to label in advance (https://asreview.readthedocs.io/en/latest/data.html). So first create your own dataset with documents retrieved from databases (https://asreview.nl/blog/the-importance-of-abstracts/), then you start asreview lab and load the dataset. Next you warm up the AI model and start labelling (in Oracle mode). While you are labelling the imported documents, asreview will learn from your labels (you train the AI on the go = active learning, see https://asreview.nl/blog/active-learning-explained/). The machine learning model will assign a relevance score to the set of unlabeled documents. If you have set the query strategy to maximum (default), asreview will show the document with the highest relevance score first. This active learning cycle is illustrated in figure 1, https://www.nature.com/articles/s42256-020-00287-7.

In the analytics tab you can keep track of how many of the imported documents you have labelled as relevant or irrelevant. You stop labelling when you think you have acquired (=labelled) most of the relevant documents in your imported dataset. Determining when to stop (=stopping rule) is difficult, as there is no golden answer available: https://asreview.nl/blog/asreview-class-101/#stop-screening.

The type of machine learning application you refer to requires a human-in-the-loop (HITL) method, where the goal is to output an accurate model in as few steps as possible, which can then be used to label unseen documents. However, because systematic reviewing requires all relevant records to be seen by the researcher and the model needs to learn a diverse range of concepts, the term researcher-in-the-loop (RITL) was introduced which has three unique components compared to HITL (https://www.nature.com/articles/s42256-020-00287-7):

1. The primary output of the process is a selection of the records, not a trained machine learning model.
2. All records in the relevant selection are seen by a human at the end of the process.
3. The use-case requires a reproducible workflow and complete transparency is required.

Does this answer your question?

**Comment by sonsuzluk0** [comment]
Thank you for your comprehensive reply!
Let's assume, I am satisfied with the number of trained/labeled documents now, I have 43, I guess. I do not see any recommendation at the moment. That is what I mean. There should be a recommendation for other documents, suggested by the algorithm.

edit: I have used the Oracle method to label 43 documents, the review is also finished.

**Reply by gimoAI** [reply]
So your imported dataset contains 43 documents right?

Asreview is not going to suggest random documents from databases/internet: you should include all documents in the dataset yourself, before you enter Oracle mode. Asreview changes the order in which you label the records in your dataset so that you are most likely to label/find the documents you are searching for first (with query strategy set to maximum).

So let's take your specific example: your current dataset contains 43 papers about dogs and cats and you only want to find papers on dogs. You have already gone through Oracle mode to assign a label to each of these 43 papers. Now you have found 100 new similar papers and you want to find the papers discussing dogs as quickly as possible.

1. Export the dataset with the 43 labelled papers:
![image](https://user-images.githubusercontent.com/65157657/189829604-e5dae47c-a513-4084-831d-b58ef620e52f.png)
2. Add the new papers you want labelled to the dataset
3. Create a new project in Oracle mode
4. Import the dataset: asreview should recognize the 43 labelled papers and add them as prior knowledge (trains the model with these), and leave the other 100 papers for you to review
5. Start reviewing the new papers

Asreview will assign a relevance score to the set of unlabeled documents (the 100 new papers). If you have set the query strategy to maximum (default), asreview will show the document with the highest relevance score (so the paper that is most likely to discuss dogs) first.

To be 100% sure you find all the papers on dogs in these 100 new papers, you will have to screen them all (saves no time). But using asreview you are likely to find most papers on dogs earlier (without having to screen all 100 new papers), saving you time!
